vendor/github.com/coredns/corefile-migration/migration/versions.go

```diff
@@ -30,7 +30,13 @@ type release struct {
 
 // Versions holds a map of plugin/option migrations per CoreDNS release (since 1.1.4)
 var Versions = map[string]release{
+	"1.11.3": {
+		priorVersion:   "1.11.1",
+		dockerImageSHA: "9caabbf6238b189a65d0d6e6ac138de60d6a1c419e5a341fbbb7c78382559c6e",
+		plugins: 		plugins_1_11_0,
```

Comment:
how do we figure out the `plugins` field value here? is it correct to be on 1.11.0 ? @BenTheElder

Reply:
The `plugins` maintained a map of plugins with deprecation status and migration actions for this release.

If no plugin deprecation status change or extra migration actions, it can use the previous version's `plugins` value.

Reply:
Thanks.. in v1.11.2 and v1.11.3 , there are good amount of changes atleast on few plugins like `forward`, thats why I was cross checking, however I don't see any specific deprecation or migration actions, so we should be good on this.